### PR TITLE
Working edits to test cases

### PR DESCRIPTION
**Pull request by drewnielson**
… placeholder. Added two new test cases.

### CODE REVIEW

File: docs/testcases/TC06-002-pass.html [hunk 1]
```diff
@@ -14,176 +14,182 @@
...
+  		<div class="row">
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="TCtitle">Image - text equivalent in alt attribute</h1>
+  							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+  								<h3 id="TCid">Test Case ID</h3>
+  									<p>TC06-002-pass</p>
+  								<h3 id="TCdesc">Test Case Description</h3>
+  									<p>Detect existence attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative output for the image. Then determine whether the text alternative output for the image provides an equivalent description of the image.</p>
```
Comment (kengdoj) [inline docs/testcases/TC06-002-pass.html:84]: Detect existence **_of_** attributes

File: docs/index.html [hunk 2]
```diff
@@ -3,100 +3,103 @@
...
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="framework-intro">ICT Baseline Alignment Framework</h1>
+  							<h2 id="purpose">Purpose and Objectives</h2>
+  								<p class="card-text">GSA's Government-wide IT Accessibility program is encouraging adoption of the Baseline more broadly across government. To facilitate adoption of the Baseline, we must create a process and framework for validating alignment of individual test processes (including testing tools) to the Baseline. The Trusted Tester process is one prime example of a test process that aligns to the Baseline. A Baseline Alignment Validation Framework will make it possible for more agencies to demonstrate alignment of their test processes (whether they subscribe to Trusted Tester or other methods) while providing broader flexibility in the selection of testing tools and contributing test methods. In the end, the more agencies that demonstrate alignment to the Baseline, the more we can share testing results and reduce redundancy in our accessibility testing and market research efforts. </p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/index.html:74]: No heartburn from me to remove reference to Trusted Tester. Done. Probably need a boarder update to this page, however, to better describe the purpose and objectives of the alignment framework/project.

File: docs/testcases/TC06-003-pass.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  		<div class="row">
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="TCtitle">Image - text equivalent in title attribute</h1>
+  							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+  								<h3 id="TCid">Test Case ID</h3>
+  									<p>TC06-003-pass</p>
+  								<h3 id="TCdesc">Test Case Description</h3>
+  									<p>Detect existence attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative output for the image. Then determine whether the text alternative output for the image provides an equivalent description of the image.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-003-pass.html:84]: fixed

File: docs/testcases/TC06-002-pass.html [hunk 4]
```diff
@@ -14,176 +14,182 @@
...
+  									<p>The code sample provides an equivalent description of the image in an alternative attribute. A successful test should identify a PASS against Baseline 6. Images.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-002-pass.html:85]: fixed

File: docs/testcases/TC06-002-pass.html (hunk 1 above)
Comment (drewnielson) [reply docs/testcases/TC06-002-pass.html:84]: fixed

File: docs/testcases/TC06-001-fail.html [hunk 5]
```diff
@@ -14,168 +14,168 @@
...
+  		<div class="row">
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="TCtitle">Image - no accessible name or description</h1>
+  							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+  								<h3 id="TCid">Test Case ID</h3>
+  									<p>TC06-001-fail</p>
+  								<h3 id="TCdesc">Test Case Description</h3>
+  									<p>Detect existence/absence of attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative output for the image.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-001-fail.html:84]: fixed

File: docs/testcases/TC06-003-pass.html [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+  									<p>The code sample provides an equivalent description of the image in an alternative attribute. A successful test should identify a PASS against Baseline 6. Images.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-003-pass.html:85]: fixed

File: docs/testcases/TC06-004-pass.html [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+  		<div class="row">
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="TCtitle">Image - text equivalent in aria-label attribute</h1>
+  							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+  								<h3 id="TCid">Test Case ID</h3>
+  									<p>TC06-004-pass</p>
+  								<h3 id="TCdesc">Test Case Description</h3>
+  									<p>Detect existence attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative output for the image. Then determine whether the text alternative output for the image provides an equivalent description of the image.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-004-pass.html:84]: fixed

File: docs/testcases/TC06-004-pass.html [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+  									  <li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html">WCAG2 SC: 1.1.1. Non-Text</a> – All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for [specific] situations listed.</li>
+  									  <li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html">WCAG2 SC: 4.1.2 Name, Role, Value</a> – For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.</li>
+  									</ul>
+  					</div>
+  				</div>
+  				<div class="card shadow my-3">
+  					<div class="card-body">
+  						<h2 class="border-bottom" id="TCdet">Test Case Detail</h2>
+  							<h3 id="TCresult">Expected Baseline Result</h3>
+  								<p><strong>FAIL</strong></p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-004-pass.html:116]: fixed

File: docs/testcases/TC06-002-pass.html [hunk 9]
```diff
@@ -14,176 +14,182 @@
...
+  									  <li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html">WCAG2 SC: 1.1.1. Non-Text</a> – All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for [specific] situations listed.</li>
+  									  <li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html">WCAG2 SC: 4.1.2 Name, Role, Value</a> – For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.</li>
+  									</ul>
+  					</div>
+  				</div>
+  				<div class="card shadow my-3">
+  					<div class="card-body">
+  						<h2 class="border-bottom" id="TCdet">Test Case Detail</h2>
+  							<h3 id="TCresult">Expected Baseline Result</h3>
+  								<p><strong>FAIL</strong></p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-002-pass.html:116]: fixed

File: docs/testcases/TC06-002-pass.html [hunk 10]
```diff
@@ -14,176 +14,182 @@
...
+  								<p>The image in the code sample does not provide any attributes that would contribute to an accessible name or description text alternative output.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-002-pass.html:117]: fixed

File: docs/testcases/TC06-004-pass.html [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+  								<p>The image in the code sample does not provide any attributes that would contribute to an accessible name or description text alternative output.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-004-pass.html:117]: fixed

File: docs/testcases/TC06-004-pass.html [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="TCtitle">Image - text equivalent in aria-label attribute</h1>
+  							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
+  								<h3 id="TCid">Test Case ID</h3>
+  									<p>TC06-004-pass</p>
+  								<h3 id="TCdesc">Test Case Description</h3>
+  									<p>Detect existence attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative output for the image. Then determine whether the text alternative output for the image provides an equivalent description of the image.</p>
+  									<p>The code sample provides an equivalent description of the image in an alternative attribute. A successful test should identify a PASS against Baseline 6. Images.</p>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-004-pass.html:85]: fixed

File: docs/testcases.html [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+  		<div class="row">
+  			<div class="col-md-9 col-xl-8 py-1 px-1 py-md-2 pl-md-3 offset-md-3 offset-xl-2 order-1">
+  				<div class="card shadow">
+  					<div class="card-body">
+  						<h1 class="card-title" id="framework-intro">ICT Baseline Alignment Framework</h1>
+  							<h2 id="purpose">Placeholder Index</h2>
+  								<p class="card-text">
+                    Until we modify the primary index file, this serves as a placeholder with links to current test case documents. </p>
+                  <p class="card-text">
+                    Test case documents are organized by Test Scenarios (aligned directly to Baseline Tests), which inculde multiple Test Cases. Will need to develop a Test Scenario template to summarize Test Scenario purpose and subordinate Test Cases.
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases.html:76]: Trying to convey the hierarchy of Test Scenario over Test Case. Basically each Baseline test serves as a Test Scenario. Revised wording to flip the reference first to Baseline Test and parenthetical reference to indicate that it serves as a Test Scenario.

File: docs/testcases/TC06-001-fail.html [hunk 14]
```diff
@@ -14,168 +14,168 @@
...
+  										<code>
   &ltbody&gt
 	&ltimg src="../assets/images/corplogo.png" width="50" height="50" class=""&gt
 	&ltp&gtThis is a meaningful image&lt/p&gt
   &lt/body&gt
+  										</code>
+  									</pre>
+  								<h4 id="TCurl">Single-Page URL</h4>
+  									<p><a href="https://section508coordinators.github.io/BaselineTestPages/testfiles/TC06-001-fail/06-001-fail.html">https://section508coordinators.github.io/BaselineTestPages/testfiles/TC06-001-fail0=/06-001-fail.html</a></p>
+  								<h4 id="TCframe">Embeded Code</h4>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-001-fail.html:130]: fixed

File: docs/testcases/TC06-001-fail.html [hunk 15]
```diff
@@ -14,168 +14,168 @@
...
+  									</tr>
+  									<tr>
+  										<td>2.</td>
+  										<td>Determine whether the image is meaningful. Review the page content and image context to determine whether the image conveys meaningful information.</td>
+  										<td>The image is meaningful, based on the text directly below the image.</td>
+  									</tr>
+  									<tr>
+  										<td>3.</td>
+  										<td>Calculate the accessible name and accessible description output for the image according to <a href="https://www.w3.org/TR/html-aam-1.0/#img-element">HTML Accessibility API Mappings 1.0 for img</a>.</td>
+  										<td>The output for the accessible name and accessible descriptions computations is <code>NULL</code>.</td>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-001-fail.html:155]: I'm good with using "empty." Fixed.

File: docs/testcases/TC06-001-fail.html [hunk 16]
```diff
@@ -14,168 +14,168 @@
...
+  										<td>One <code>&ltimg&gt</code> exists in the test data code sample.</td>
+  									</tr>
+  									<tr>
+  										<td>2.</td>
+  										<td>Determine whether the image is meaningful. Review the page content and image context to determine whether the image conveys meaningful information.</td>
+  										<td>The image is meaningful, based on the text directly below the image.</td>
+  									</tr>
+  									<tr>
+  										<td>3.</td>
+  										<td>Calculate the accessible name and accessible description output for the image according to <a href="https://www.w3.org/TR/html-aam-1.0/#img-element">HTML Accessibility API Mappings 1.0 for img</a>.</td>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-001-fail.html:154]: Hmmm - I think agree, but let's discuss. I've added an issue in the Baseline repository for consideration.

File: docs/testcases/TC06-001-fail.html [hunk 17]
```diff
@@ -14,168 +14,168 @@
...
+  										<td>The output for the accessible name and accessible descriptions computations is <code>NULL</code>.</td>
+  									</tr>
+  									<td>4.</td>
+  										<td>Determine whether any checks fail according to the <a href="#test-instructions">6. Images test instructions</a>.</td>
+  										<td>The first check fails due to absence of accessible name or description in the text alternative computation output.</td>
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/testcases/TC06-001-fail.html:159]: fixed

**Comment by drewnielson** [comment]
@kengdoj thanks so much for your detailed review! I made edits to address your comments. I resolved the comments that I fixed. I left a couple comments unresolved so you can review whether you like the change. Resolve those comments if satisfied and approve the pull request if ready to merge to master.

**Pull request merged**